README.md

```diff
@@ -46,6 +46,38 @@ The site's default CSS has now moved to a new place within the gem itself, [`ass
 
 --
 
+When you override only a minima-sass-partial, it is not automatically imported because we're still importing the `minima.scss` within the theme-gem and that subsequently imports the partials with respect to itself, i.e. partials within the gem. Hence you should either include a *copy of `minima.scss` from the gem* inside the `_sass` directory at source or the overriding `/assets/main.scss` file should explicitly import the edited partial. :
+  e.g. To have an **edited** `_syntax-highlighting.scss` be rendered, you should either have
+
+```sass
+/* <your-site>/assets/main.scss */
+
+@import "minima";
+@import "minima/syntax-highlighting";
```

Comment:
Maybe I am not understanding.  I read it as you you are presenting two different ways to accomplish the goal of overriding a minima-sass-partial.  One way you duplicate a file from the gem, the other you explicitly import your modifications.  The later way does not require the user to maintain a copy of _minima.scss_ making it more robust and less likely to break.  I was suggesting that be the approach that is presented to users and that the other one perhaps should not be.

Context: I encountered this exact issue first-hand with [json-schema.org](https://github.com/json-schema-org/json-schema-org.github.io).  On my own I figured out your first way of including a copy of `minima.scss`.  This worked, but then building the site broke when GitHub updated minima to v2.1.1 and `minima.scss` needed up be updated.  This would not have happened if I knew to just import the modified scss file.